lib/jenkins/retry.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+#
+# Copyright (C) 2015-2017 Harald Sitter <sitter@kde.org>
+# Copyright (C) 2018 Jonathan Riddell <jr@jriddell.org>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) version 3, or any
+# later version accepted by the membership of KDE e.V. (or its
+# successor approved by the membership of KDE e.V.), which shall
+# act as a proxy defined in Section 6 of version 3 of the license.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+require_relative '../../ci-tooling/lib/jenkins'
+
+module Jenkins
+
+  def retry_jobs(job_name_queue, exclusion_states=%w[success unstable], strict_mode=false)
```

Comment:
method too long

all arguments but the queue should be keyword arguments

(also strict_mode is a really bad name, could be called foobar and it'd be just as indicative of what it does)